src/colors.js

```diff
@@ -2,6 +2,6 @@ export const primary_text = '#fff';
 export const card_title = '#3c3c3c';
 export const card_subtext = '#707275';
 
-export const unseleted_button = '#3d917b';
+export const unselected_button = '#3d917b';
```

Comment:
Don't love the naming of these. Or the casing. Sorry I missed this before, but they should be camel cased and I'd like to have something like "primary" rather than what they are being used on.